common/backoff/cron.go

```diff
@@ -45,7 +46,12 @@ func ValidateSchedule(cronSchedule string) error {
 
 // GetBackoffForNextSchedule calculates the backoff time for the next run given
 // a cronSchedule, workflow start time and workflow close time
-func GetBackoffForNextSchedule(cronSchedule string, startTime time.Time, closeTime time.Time) time.Duration {
+func GetBackoffForNextSchedule(
+	cronSchedule string,
+	startTime time.Time,
+	closeTime time.Time,
+	jitterStartSeconds int32,
+) time.Duration {
```

Comment:
if you are moving the jitter to this function, you might want to change the return type to `(error, time.Duration)`